Question_10.py

```diff
@@ -0,0 +1,103 @@
+'''
+Expression-1:
+**_**
+*___*
+_____ 
+*___*
+**_**
+
+'''
+print("Expression-1")
+global k 
+n = int(input("Enter any number to dispaly Expression-1 :-> "))
+n = n//2 + 1 if n%2!=0 else n//2
+k = 1
+for j in range(n,0,-1):
+    print(j*'*' + k*'-' + j*'*')
+    k += 2
+
+k = k-4
+for j in range(2,n+1):
+    print(j*'*' + k*'-' + j*'*')
+    k -= 2
+
+
+'''
+Expression - 2:
+__*__ 
+_***_ 
+***** 
+_***_ 
+__*__
+'''
+print("\nExpression-2")
+n = int(input("Enter any number to dispaly Expression-2 :-> "))
+
+n = n//2 + 1 if n%2!=0 else n//2
+k = 1
+for j in range(n,0,-1):
+    print(j*'-' + k*'*' + j*'-')
+    k += 2
+
+k = k-4
```

Comment:
It seems like Patiya.

Reply:
> It seems like Patiya.

Sir, I have used it to adjust the lower triangle pattern and it will work for any positive natural number.